core/storage/user/gae_models_test.py

```diff
@@ -44,6 +44,51 @@ def test_get_by_role(self):
             feconf.ROLE_ID_ADMIN)
         self.assertEqual(user[0].role, feconf.ROLE_ID_ADMIN)
 
+    def test_export_data(self):
```

Comment:
We should also have a test for this where all field values are nontrivial (see my similar comment on the other model).